lib/main.dart

```diff
@@ -28,23 +28,11 @@ class MyApp extends StatelessWidget {
                 '/login': (context) => LoginPage(),
                 '/feed': (context) => FeedPage(),
               },
-              home: Wrapper(),
+              home: LoginPage(),
```

Comment:
why you changing this ?

Reply:
for previewing login page

Reply:
mera code works too.. dont change this
```suggestion
              home: Wrapper(),
```